go_modules/spec/dependabot/go_modules/file_updater_spec.rb

```diff
@@ -127,6 +127,15 @@ module declares its path as: go.etcd.io/bbolt
       end
     end
 
+    context "pre 1.21 go.mod that uses a 1.21 dependency" do
+      let(:project_name) { "toolchain" }
+      let(:files) { [go_mod] }
+
+      it "doesn't add a toolchain directive" do
+        expect(updated_files.first&.content).to_not include("toolchain")
```

Comment:
🔎 I think we should omit `&.` here as it will pass `nil` to the `include` matcher. I think the resulting `NoMethodError` for `#content` would be a little easier to reason about as it's closer to the code.

```suggestion
        expect(updated_files.first.content).to_not include("toolchain")
```